deploy/cephcsi/image/Dockerfile

```diff
@@ -1,15 +1,50 @@
+ARG SRC_DIR="/go/src/github.com/ceph/ceph-csi/"
```

Comment:
Probably better to move this `ARG` to the others further below?

Reply:
this ARG is required for both stages, defined it at top

Reply:
Oh, I was not aware it matters for `ARG` to be in a specific location.